PRINCIPLES.md

```diff
@@ -9,12 +9,20 @@ core geospatial standards.
 The collaboration facilities of Github should be used to their full extent. All proposed improvements and
 changes should come in the form of pull requests, using code review functionality to discuss changes.
 
-- **JSON + REST + HTTP at the core.** JSON has won over XML, and REST over SOAP. We embrace them and
+- **Alignment with OGC standards** - The Open Geospatial Consortium publishes a large set of geospatial standards.
+  To the greatest extent possible, the STAC API should align with existing and in-progress OGC API standards. The
+  STAC API has a symbiotic relationship with these standards, as it seeks both to reuse their building blocks and
+  push them forward in a practical direction. Among the most important of these are:
+  - [OGC API - Features](https://ogcapi.ogc.org/features/), particularly [OGC API - Features - Part 1: Core](http://docs.ogc.org/is/17-069r3/17-069r3.html)
+  - [OGC API - Records](https://ogcapi.ogc.org/records/)
```

Comment:
Would it be good to also mention OGC API - Commons?

Reply:
done.